.gitignore

```diff
@@ -2,6 +2,9 @@
 /node_modules
 /bin
 
+# git-crypt
+tegGenKey
```

Comment:
the key file should not be in the directory, so i don't think this is necessary. Usually you only want to share this with the developers and they store it where they want, a lot of people have a separate folder with keys for different projects